### ISSUE TITLE
Reduce image sizes

### ISSUE THREAD

**Issue by benjaoming** [mirroringplaces/mirroringplaces.online#1]
@johanne-aarup-hansen kan vi gøre noget ved billedfilernes størrelse? De er meget meget store. Jeg synes godt, at websitet kan leve med at have lidt store billeder; og vi kan evt. lege lidt med at få websitet til at virke, inden de er indlæste.

![image](https://github.com/mirroringplaces/mirroringplaces.online/assets/374612/4f6e54eb-d5f4-4056-a5f9-b4c509299670)

**Comment by johanne-aarup-hansen** [comment]
Ja, vi kan godt gøre dem mindre, men de skal heller ikke alle sammen bruges, så vi kan også bare slette dem vi ikke bruger når det er

**Comment by benjaoming** [comment]
Yes, lad os rydde lidt op sammen :+1: 

Ift. load-hastighed, så har Firefox en funktion, hvor du kan simulere at bruge en langsommere forbindelse. Det er en smart måde hurtigt at finde ud af, om man har lavet et website, der er for tungt.

![image](https://github.com/mirroringplaces/mirroringplaces.online/assets/374612/030995d4-bef8-4c11-b0b1-386a4d5bad02)

Jeg tror at load-tider omkring 2-3 sekunder er "forventeligt", men det er self. anderledes ift. websitets formål. Når der er meget grafik, vil den almindelige bruger forhåbentligt forstå pointen med load-tiden :)

**Comment by johanne-aarup-hansen** [comment]
I have made a pull request about this... should I have done that here some how?

**Comment by benjaoming** [comment]
I think the state of the background images is good now!
